README.md

```diff
@@ -44,6 +44,9 @@ package.dependencies += [
 
 ```ruby
 pod 'CombineX', "~> 0.3.1"
+
+# or, if you want to use `Foundation` extensions: 
+pod 'CombineX/CXFoundation', "~> 0.3.1"
```

Comment:
I have very little knowledge of CocoaPods. So we don't need to state CXFoundation explicitly before, but now we do? With duplicate version number?

Reply:
It's an either-or

`pod 'CombineX'` means you only want `CombineX`.
`pod 'CombineX/CXFoundation` means you want  `CombineX`, and `CXFoundation`.

Reply:
Did users need `pod 'CombineX/CXFoundation'` before?

Reply:
Yes, if they want to use `CXFoundation`